region/maridia/outer/Mt. Everest.json

```diff
@@ -941,6 +941,35 @@
         "In the likely event of failure, the room will need to be reset and the crabs repositioned for the next attempt."
       ]
     },
+    {
+      "link": [2, 7],
+      "name": "Scisor Double Damage Boost",
+      "requires": [
+        "h_canNavigateUnderwater",
+        "canCrouchJump",
+        "canInsaneJump",
+        "canReserveDoubleDamageBoost",
+        {"or": [
+          {"and": [
+            "canStationaryLateralMidAirMorph",
+            "canTrickySpringBallJump"
```

Comment:
I ended up only testing one water location (Main St with HJB and Morph), but plan to test one dry location as well as the jump height without damage boosting in each of these locations to make sure how similar everything is. I was curious though why this one needed `canStationaryLateralMidAirMorph` but the Main Street one didn't?
```suggestion
            "h_canMaxHeightSpringBallJump"
```

Reply:
The difference is in the terrain; main street is flat with a slope on the side and everest has a gentle pit below where you want to jump.  And you need to be against the right wall when you hit the crab.

- In main street I'm finding jumping from the slope works best
- In everest I was jumping from the left of the crab to hit it from above, but maybe its easier to jump from the right and hit it from below like how I'm doing main street.